spec/unit/proxy_spec.rb

```diff
@@ -1,15 +1,13 @@
-require "spec_helper"
-
 module Spank
   describe Proxy do
-    let(:sut) { Proxy.new(target) }
+    subject { Proxy.new(target) }
     let(:target) { double("target", :greet => nil) }
 
     context "when invoking a method" do
-      before { sut.greet('blah') }
+      before { subject.greet('blah') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.